### DISCUSSION TITLE
Welcome to imdbpy Discussions!

### DISCUSSION THREAD

**Discussion by alberanid** [cinemagoer/cinemagoer#337] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BradKML** [comment]
Currently wanting to trace the relationship between people's involvement in older TV shows and how it leads to being the head of newer TV shows. Also possibly see which actor or director makes the most impact based on PageRank (social network) or CiteRank (DAG and referencing).

**Reply by BradKML** [reply]
Eyeing
- https://github.com/networkx/networkx/discussions/5062
- https://github.com/networkx/networkx/discussions/4674
- https://github.com/igraph/igraph/issues/1741
- https://github.com/biolab/orange3-network/issues/235
- https://github.com/StephanyRajeh/ModularCentralities/issues/1
